content/millennium/r4/scheduling/appointment.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Appointment | R4 API
+---
+
+# Appointment
+
+* TOC
+{:toc}
+
+## Overview
+
+The Appointment resource provides the ability to retrieve information about appointments, write new appointments, and update the status of existing appointments.
+
+Date is required when searching by patient, practitioner, or location.
+
+When integrating your application with a client’s production environment you will work with the client to determine the Practitioner and Location ids (Millennium personnel and location codes, respectively) which they want to make available to third-party applications for enabling scheduling functionality.
+
+When updating an appointment, the resource provides the ability to change the Appointment.status from booked to arrived, checked-in, or cancelled.
+
```

Comment:
booked -> arrived, checked-in, cancelled
arrived -> checked-in, cancelled
checked-in -> cancelled
But from cancelled we can't move to any other state I guess.
Not sure if it will be clear for a person reading this doc.

Reply:
+1 to @kchakdar - I would word it like:

When updating an appointment, the resource provides the ability to change the Appointment.status in the following sequences:
- From Booked to Arrived, Checked-In, or Cancelled
- From Arrived to Checked-In, or Cancelled
- From Checked-In to Cancelled